robot/description/urdf/caffeine.gazebo.xacro

```diff
@@ -93,5 +93,56 @@
             <!-- TODO: add Gaussian noise based off Garmin GPS -->
         </plugin>
     </gazebo>
+    <!-- TODO: double check these numbers -->
```

Comment:
Some of the numbers we are looking for can be found here: https://www.stereolabs.com/zed/. For example, for an image resolution of 720p (1280x720) the camera runs at a rate of 60 Hz (ours currently uses 30 Hz).

Additionally, all values in tags should exist as constants in `constants/xacro` (see how all the other plugins use constants)